spec/system/admin/order_cycles/edit_spec.rb

```diff
@@ -0,0 +1,91 @@
+# frozen_string_literal: true
+
+require 'system_helper'
+
+RSpec.describe '
+    As an administrator
+    I want to edit a specific order cycle
+' do
+  include AdminHelper
+  include AuthenticationHelper
+  include WebHelper
+
+  let(:oc0) {
+    create(:simple_order_cycle, name: 'oc0',
+                                orders_open_at: nil, orders_close_at: nil)
+  }
+  let(:oc1) { create(:order_cycle, name: 'oc1') }
+
+  context 'when cycle has attached schedule(s)' do
+    it "properly toggles order cycle save bar buttons to show warning modal" do
+      create(:schedule, name: 'Schedule1', order_cycles: [oc0])
+
+      # When I go to the admin order cycle edit page
+      login_as_admin
+      visit edit_admin_order_cycle_path(oc0)
+
+      expect(page).to have_selector("#linked-schedule-warning-modal")
+      expect(page).not_to have_selector("#modal-actions")
+      expect(page).to have_selector("#form-actions")
```

Comment:
Where possible, we prefer to check for things that the user can see.
In this case, I think these expectations ☝️  aren't necessary.